src/wizard/Step1/components/PermissionsModal.vue

```diff
@@ -1,11 +1,181 @@
+            Email address
+          </ATATTextField>
+          <br />
+          <v-btn class="link-body-md" :ripple="false">
+            Add another portfolio manager
+          </v-btn>
+          <br /><br />
+          <h3 class="h3">Portfolio Permissions</h3>
+          <div class="permissions-set-list">
+            <div class="permission-set-checkbox">
+              <v-checkbox
+                class="ma-0 pa-0"
+                :ripple="false"
+                label="Edit Funding"
+              />
+              <p>Can add or modify Task Orders to fund this Portfolio</p>
+            </div>
+            <div class="permission-set-checkbox">
+              <v-checkbox
+                class="ma-0 pa-0"
+                :ripple="false"
+                label="Edit Funding"
+              />
+              <p>Can add or modify Task Orders to fund this Portfolio</p>
+            </div>
+            <div class="permission-set-checkbox">
+              <v-checkbox
+                class="ma-0 pa-0"
+                :ripple="false"
+                label="Edit Funding"
+              />
+              <p>Can add or modify Task Orders to fund this Portfolio</p>
+            </div>
+            <div class="permission-set-checkbox">
+              <v-checkbox
+                class="ma-0 pa-0"
+                :ripple="false"
+                label="Edit Funding"
+              />
+              <p>Can add or modify Task Orders to fund this Portfolio</p>
+            </div>
+          </div>
+        </v-card-text>
+        <v-card-actions class="d-flex justify-end">
+          <v-btn color="primary" text @click="doCancel()"> Close </v-btn>
+          <v-btn color="primary" @click="doSave()"> Send Invitation </v-btn>
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
+  </div>
 </template>
 
 <script lang="ts">
 import Vue from "vue";
-import { Component } from "vue-property-decorator";
+import { Component, Emit, Prop } from "vue-property-decorator";
+import ATATTextField from "@/components/ATATTextField.vue";
 
-@Component({})
-export default class PermissionsModal extends Vue {}
+export interface KeyValuePair {
+  (key: string): any;
+}
+
+export interface ActionObject {
+  action: string;
+  data: KeyValuePair[];
+}
+
+@Component({
+  components: { ATATTextField },
+})
+export default class PermissionsModal extends Vue {
+  @Prop({ default: true }) private isDialogVisible!: boolean;
+
+  @Emit()
+  private modalAction(action: string, data: KeyValuePair[] = []): ActionObject {
+    console.log("clickedAction in");
+    return {
+      action,
+      data,
+    };
+  }
+
+  private doCancel(): void {
+    // do something
+    // emit action and object
+    console.log("doCancel");
+    this.modalAction("cancel");
+  }
+  private doSave(): void {
+    // do something
+    // emit action and object
+    console.log("doSave");
+    this.modalAction("save");
+  }
+}
 </script>
+
+<style lang="scss">
+.portfolio-managers-modal-dialog {
```

Comment:
make sure use as many out-of-the-box veutify classes that you can like for padding and display and position and height.

- [Padding and margin](https://vuetifyjs.com/en/styles/spacing/#negative-margin)
- [display helpers](https://vuetifyjs.com/en/styles/display/#accessibility)
- [Flex](https://vuetifyjs.com/en/styles/flex/)
- [Float] (https://vuetifyjs.com/en/styles/float/#overview)